### PR TITLE
Feature/gds select workplace address page

### PR DESCRIPTION
**Pull request by omarCreativeDev**
- the `select workplace address` screen has now been split in to 2 screens as per rich's latest designs, the change postcode logic will be on a new route called `find-workplace-address` which is also included in this pr
- I'm going to work on `enter-workplace-address.component` in my next pr so please ignore the comment out code for now

### CODE REVIEW

**Comment by omarCreativeDev** [comment]
[select workplace address page](https://trello.com/c/p2CCGb8R/550-select-workplace-address-page)

**Comment by omarCreativeDev** [comment]
[find workplace address page](https://trello.com/c/m5R0GU3o/558-find-workplace-address-page)

File: src/app/features/registration/enter-workplace-address/enter-workplace-address.component.ts [hunk 1]
```diff
@@ -266,22 +260,22 @@ export class EnterWorkplaceAddressComponent implements OnInit {
...
-      this.enterWorkplaceAddressForm.setValue({
-        postcodeInput: postcodeValue,
-        address1Input: address1Value,
-        address2Input: address2Value,
-        townCityInput: townCityValue,
-        countyInput: countyValue,
-        wpNameInput: '',
-      });
-    }
+    // if ((this.registration.locationdata[0].hasOwnProperty('locationName')) && (this.registration.locationdata[0].locationName === '')) {
```
Comment (ChazUK) [inline src/app/features/registration/enter-workplace-address/enter-workplace-address.component.ts]: Guessing this is to be removed in a future PR?

Comment (omarCreativeDev) [reply src/app/features/registration/enter-workplace-address/enter-workplace-address.component.ts]: yup im working on enter-workplace-address as we speak

**Pull request merged**